src/com/amazon/ion/benchmark/IonUtilities.java

```diff
@@ -419,8 +447,9 @@ static void rewriteIonFile(Path input, Path output, OptionsCombinationBase optio
                 options.importsForInputFile == null &&
                 options.importsForBenchmarkFile == null &&
                 options.format == Format.ION_BINARY &&
-                options.ionMinorVersion != 1 && // TODO remove once support for writing system values via the Ion 1.1 writer is added.
-                IonUtilities.minorVersionsEqual(options.ionMinorVersion, input.toFile())
+                // Minor versions may add new kinds of system values, so it is not possible to maintain system value
```

Comment:
Minor versions could also add new symbols to the system symbol table. Do we need to account for that somehow to make sure that the transcoded SIDs still point to the correct text?

Reply:
Oof, yeah... looking into whether this is feasible.

Reply:
Proposed update to ion-java to enable this here: https://github.com/amazon-ion/ion-java/pull/876
And I updated this PR to perform the transcoding. We'll have to make a minor update to ion-java once the 1.1 system symbol table is finalized.